src/data/fintype/sort.lean

```diff
@@ -17,3 +17,22 @@ casting issues in further uses of this function. -/
 def mono_equiv_of_fin (α) [fintype α] [linear_order α] {k : ℕ}
   (h : fintype.card α = k) : fin k ≃o α :=
 (univ.order_iso_of_fin h).trans $ (order_iso.set_congr _ _ coe_univ).trans order_iso.set.univ
+
+variables [decidable_eq α] [fintype α] [linear_order α] {m n : ℕ}
+  {s : finset α} [decidable_pred (s : set α)]
+
+/-- If `α` is a linearly ordered type, `s : finset α` has cardinality `m` and its complement has
+cardinality `n`, then `fin m ⊕ fin n ≃ α`. -/
```

Comment:
Since there are multiple equivalences that could exist here, could you expand in the docstring which one is used?

Reply:
I expanded the docstring. Feel free to edit it.